src/nft/components/explore/CarouselCard.tsx

```diff
@@ -13,13 +13,32 @@ const CarouselCardContainer = styled.div`
   gap: 20px;
   background-color: ${({ theme }) => theme.backgroundSurface};
   border-radius: 20px;
-  outline: 1px solid ${({ theme }) => theme.backgroundOutline};
+  overflow: hidden;
+`
+const CarouselCardBorder = styled.div`
   width: 100%;
+  position: relative;
+  border-radius: 20px;
   cursor: pointer;
-  overflow: hidden;
 
-  :hover {
-    outline: 3px solid ${({ theme }) => theme.backgroundOutline};
+  ::before {
+    content: ' ';
+    position: absolute;
+    top: -1px;
+    right: -1px;
+    bottom: -1px;
+    left: -1px;
+    border-radius: 21px;
+    border: 1px solid ${({ theme }) => theme.backgroundOutline};
+  }
+
+  :hover::before {
+    top: -3px;
+    right: -3px;
+    bottom: -3px;
+    left: -3px;
+    border-radius: 23px;
```

Comment:
is the 21 -> 23px change on hover intentional?

Reply:
Yes, this is needed since the border on hover is 3px wide, which requires more radius to align with the inner element.